src/Hateoas/Configuration/Relation.php

```diff
@@ -42,7 +42,7 @@ class Relation
     public function __construct($name, $href = null, $embedded = null, array $attributes = array(), Exclusion $exclusion = null)
     {
         if (null !== $embedded && !$embedded instanceof Embedded) {
-            $embedded = new Embedded($embedded);
+            $embedded = new Embedded($embedded, null, $exclusion);
```

Comment:
I really don't understand this patch.

If you want to specify an exclusion for the embedded, then just instantiate the embedded yourself:
```php
$relation = new Relation('name', '/href', new Embedded($data, null, new Exclusion()))
```